src/settings/Settings.tsx

```diff
@@ -891,6 +891,12 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         default: false,
         controller: new ReloadOnChangeController(),
     },
+    "automaticKeyBackNotEnabledReporting": {
+        displayName: _td("Automatically send debug logs when key backup is not functioning"),
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
+        default: false,
+        controller: new ReloadOnChangeController(),
```

Comment:
Toggling that setting probably does not warrant a whole app restart?
At least there's nothing  I can spot here that would warrant that, but might have missed it

```suggestion
```